app/views/tweets/new.erb

```diff
@@ -0,0 +1,4 @@
+<form action="/tweets/new" method="POST">
```

Comment:
RESTful convention for post routes should go to just the plural of the model, in this case, `POST '/tweets'`